airbyte-integrations/connectors/source-salesforce/source_salesforce/api.py

```diff
@@ -154,6 +154,30 @@
     "TaskStatus",
     "TaskWhoRelation",
     "UndecidedEventRelation",
+    "WorkOrderLineItemStatus",
+    "WorkOrderStatus",
+    "UserRecordAccess",
+    "OwnedContentDocument",
+    "OpenActivity",
+    "NoteAndAttachment",
+    "Name",
+    "LookedUpFromActivity",
+    "FolderedContentDocument",
+    "ContractStatus",
+    "ContentFolderItem",
+    "CombinedAttachment",
+    "CaseTeamTemplateRecord",
+    "CaseTeamTemplateMember",
+    "CaseTeamTemplate",
+    "CaseTeamRole",
+    "CaseTeamMember",
+    "AttachedContentDocument",
+    "AggregateResult",
+    "AccountHistory",
```

Comment:
Hi!
Since this connector version I'm having trouble getting data from `AccountHistory` table. The table has been added to 'UNSUPPORTED_BULK_API_SALESFORCE_OBJECTS' list.

Airbyte doesn't fail the sync job even though I'm not receiving data and find this in my log
```
2022-11-15 17:45:06 �[44msource�[0m > Syncing stream: AccountHistory 
2022-11-15 17:47:07 �[44msource�[0m > [{"message":"Your query request was running for too long.","errorCode":"QUERY_TIMEOUT"}]
2022-11-15 17:47:07 �[44msource�[0m > Cannot receive data for stream 'AccountHistory', error message: 'Your query request was running for too long.'
2022-11-15 17:47:07 �[44msource�[0m > Read 0 records from AccountHistory stream
2022-11-15 17:47:07 �[44msource�[0m > Finished syncing AccountHistory
```

After downgrading the Salesforce connector from 1.0.26 to the prior version 1.0.25 I'm able to get data from `AccountHistory` again.

Some context to that error: https://github.com/airbytehq/airbyte/issues/17503

Reply:
hi @PierreKerschgens thanks for the feedback! I prepared a PR to fix this: https://github.com/airbytehq/airbyte/pull/19869